spacemacs

```diff
@@ -41,14 +42,18 @@ values."
      better-defaults
      emacs-lisp
      git
+     yaml
+     html
+     json
+     prettier
      markdown
      javascript
      typescript
      python
      go
-     spell-checking
      syntax-checking
      version-control
+     wakatime
```

Comment:
this isn't working since we need to setup data from local wakatime config.

Tracking this Issue in #5